lib/roo_on_rails/railties/http.rb

```diff
@@ -0,0 +1,31 @@
+module RooOnRails
+  module Railties
+    class HTTP < Rails::Railtie
+      initializer 'roo_on_rails.http' do |app|
+        $stderr.puts 'initializer roo_on_rails.http'
+        require "rack/timeout/base"
+        require "roo_on_rails/rack/safe_timeouts"
+
+        ::Rack::Timeout.service_timeout = ENV.fetch('RACK_SERVICE_TIMEOUT', 15).to_i
+        ::Rack::Timeout.wait_timeout = ENV.fetch('RACK_WAIT_TIMEOUT', 30).to_i
+        ::Rack::Timeout::Logger.level = Logger::WARN
+
+        Rails.application.config.middleware.insert_before(
+          ::Rack::Runtime,
+          ::Rack::Timeout
+        )
+
+        # This needs to be inserted low in the stack, before Rails returns the
+        # thread-current connection to the pool.
+        app.config.middleware.insert_before(
+          ActionDispatch::Cookies,
+          RooOnRails::Rack::SafeTimeouts
+        )
+
+        app.config.middleware.use ::Rack::Deflater
+
+        app.config.use_ssl = true unless Rails.env.test?
```

Comment:
I've seen `use_ssl` before but I can't find it documented anywhere.
I think [`force_ssl`](http://blog.bigbinary.com/2016/08/24/rails-5-adds-more-control-to-fine-tuning-ssl-usage.html) is what you're aiming fore here, though.

I tend to use [`rack-ssl-enforcer`](https://github.com/tobmatth/rack-ssl-enforcer) as it's well-tested, documented, not Rails-dependent and clear on what it does, but up to you.

Either way - this isn't test-covered; sorry I missed that during the first review.